### DISCUSSION TITLE
React/data-fetching-suspense/

### DISCUSSION THREAD

**Discussion by giscus** [yeopto/yeopto.github.io#17] [Comments]
# React/data-fetching-suspense/

React에서 data fetching 방법론을 통해 suspense를 알아보자

https://yeopto.github.io/React/data-fetching-suspense/

**Comment by Gilpop8663** [comment]
엽토의 글을 보고 Suspense에 대한 생각을 정리할 수 있었어요! 글 공유해주셔서 감사합니다

특히 패키지를 사용하지 않고 promise 혹은 error을 던지는 부분을 어떻게 직접 구현할까 라는 생각이 있었는데 답을 얻고 갑니다~ 👍👍👍
